src/app.py

```diff
@@ -105,23 +111,29 @@ def convert_string_to_list(string):
     dbc.Row([
         dbc.Col([
             html.Div([
-                html.H1('Spotify Popularity Dashboard'),
+                html.H1('Spotify', style={'color':'white','align-items': 'left','margin-left': '30px'}),
+                html.H1('Popularity', style={'color':'white', 'align-items': 'left','margin-left': '30px'}),
+                html.H1('Dashboard', style={'color':'white', 'align-items': 'left','margin-left': '30px'}),
                 html.P("This dashboard is designed for helping record companies to make data driven decisions, so that they can provide valuable and actionable suggestions that can be used as guidance for artists aiming to enhance their music's appeal.",
-                    style={"font-size": "16px"}),
+                    style={"font-size": "16px",'color':'grey', 'margin-left': '15px'}),
```

Comment:
Looks great! My only comment is that maybe we could make this a lighter grey or white since it's a bit hard to see (but I can also do this in final touches if that's easier)!